.github/workflows/type_annotation.yml

```diff
@@ -14,9 +14,9 @@ jobs:
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so that typilus can access it.
     - uses: actions/checkout@v2
-    - uses: typilus/typilus-action@master
+    - uses: typilus/typilus-action@main
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        # MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
+        MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
```

Comment:
```suggestion
```
There is no saved model in the repo for now. All are created via scripts for now